docs/site/.helm/values.yaml

```
@@ -11,11 +11,23 @@ certificateClusterIssuer:
   _default: letsencrypt
   prod-sel: letsencrypt-nginx-tfprod
 
+moduleWatcher:
+  registries:
+    _default:
+      - registry.deckhouse.io/deckhouse/fe/modules
+      - registry.flant.com/team/foxtrot/parca
+      - registry.flant.com/team/foxtrot/docs-example
```

```suggestion
      - registry.deckhouse.io/deckhouse/fe/modules
      - registry.flant.com/team/foxtrot/docs-example
```

I guess we can remove this `parca`